docs/user_guide/models/multivi.rst

```diff
@@ -46,9 +46,7 @@ MultiVI posits that the observed UMI counts for cell :math:`n`, gene :math:`g`,
    :nowrap:
 
    \begin{align}
-    z^{acc}_n &\sim {\mathrm{Normal}}\left( {0,I} \right) \\
-    z^{rna}_n &\sim {\mathrm{Normal}}\left( {0,I} \right) \\
-    z_n &= \frac{\left( z^{rna}_n + z^{acc}_n \right)}{2} \\
+    z_n &= &\sim {\mathrm{Normal}}\left( {0,I} \right) \\
```

Comment:
I liked it better with the z acc and z rna and the averaging and then the removed paragraph before. It's still a bit amiguous when one modality is missing, you could introduce a mask?

Reply:
I believe this is not accurate.

Reply:
```suggestion
    z_n &\sim {\mathrm{Normal}}\left( {0,I} \right) \\
```